# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1284576594 14400
#      Wed Sep 15 14:49:54 2010 -0400
# Node ID 1ec2f19857fa8c8cfd1b1ba89c477ddb88002195
# Parent  5360734281823edc804f208ebc3a928406adc3b4
fix typo in previous change to load-path.cc

diff --git a/src/load-path.cc b/src/load-path.cc
--- a/src/load-path.cc
+++ b/src/load-path.cc
@@ -2061,17 +2061,17 @@ DEFUN (addpath, args, nargout,
 @deftypefnx {Built-in Function} {} addpath (@var{dir1}, @dots{}, @var{option})\n\
 Add @var{dir1}, @dots{} to the current function search path.  If\n\
 @var{option} is @samp{\"-begin\"} or 0 (the default), prepend the\n\
 directory name to the current path.  If @var{option} is @samp{\"-end\"}\n\
 or 1, append the directory name to the current path.\n\
 Directories added to the path must exist.\n\
 \n\
 In addition to accepting individual directory arguments, lists of\n\
-directory names separated by @code{pathsep} are also accepted.  For example,\n
+directory names separated by @code{pathsep} are also accepted.  For example,\n\
 \n\
 @example\n\
 addpath (\"dir1:/dir2:~/dir3\");\n\
 @end example\n\
 @seealso{path, rmpath, genpath, pathdef, savepath, pathsep}\n\
 @end deftypefn")
 {
   octave_value retval;
@@ -2172,17 +2172,17 @@ addpath (\"dir1:/dir2:~/dir3\");\n\
 }
 
 DEFUN (rmpath, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} rmpath (@var{dir1}, @dots{})\n\
 Remove @var{dir1}, @dots{} from the current function search path.\n\
 \n\
 In addition to accepting individual directory arguments, lists of\n\
-directory names separated by @code{pathsep} are also accepted.  For example,\n
+directory names separated by @code{pathsep} are also accepted.  For example,\n\
 \n\
 @example\n\
 rmpath (\"dir1:/dir2:~/dir3\");\n\
 @end example\n\
 @seealso{path, addpath, genpath, pathdef, savepath, pathsep}\n\
 @end deftypefn")
 {
   // Originally by Etienne Grossmann. Heavily modified and translated
